uclient-fetch.c

```diff
@@ -549,11 +584,11 @@ static const struct option longopts[] = {
 	[L_PROXY] = { "proxy", required_argument, NULL, 0 },
 	[L_NO_PROXY] = { "no-proxy", no_argument, NULL, 0 },
 	[L_QUIET] = { "quiet", no_argument, NULL, 0 },
+	[L_HEADER] = { "header", required_argument, NULL, 0 },
```

Comment:
IMHO the better to place the L_HEADER after the L_USER_AGENT

Reply:
Up to now this list has been appended chronologically rather than in semantic or alphabetic order. I basically just tried to keep the existing style without causing more work -- I agree though that semantic or alphabetic order would be much nicer, I was just too lazy to have another bunch of commits before the change I desire doing all the cleanup...